### PR TITLE
Explicitly use Rules.xml file to filter alpha/beta versions

### PR DESCRIPTION
**Pull request by fdeschodt**
Without this, the file was not take into account

### CODE REVIEW

**Comment by fredboutin** [comment]
> Without this, the file was not take into account

Are you sure it's always the case? I remember adding this file and seeing a different behaviour from lure

File: lib/lure/mvn.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func mvnOutdated(path string) (error, []moduleVersion) {
-	cmd := exec.Command("mvn", "versions:display-dependency-updates", "-DprocessDependencyManagement=false")
+	cmd := exec.Command("mvn", "versions:display-dependency-updates", "-DprocessDependencyManagement=false", "-Dmaven.version.rules=file:Rules.xml")
```
Comment (fredboutin) [inline lib/lure/mvn.go:20]: what happens if the file does not exist?

**Comment by fdeschodt** [comment]
> > Without this, the file was not take into account
> 
> Are you sure it's always the case? I remember adding this file and seeing a different behaviour from lure

All I can say is that it is the only way I got it to work. It wasn't working for Jake either, but this always worked

File: lib/lure/mvn.go (hunk 1 above)
Comment (fdeschodt) [reply lib/lure/mvn.go:20]: Nothing. You get exactly the same output as if you don't have the parameter

**Review by fredboutin — APPROVED** [review]

**Review by AndyCoveo — APPROVED** [review]

**Pull request merged**